arena.js

```diff
@@ -0,0 +1,69 @@
+class Arena {
+  constructor (name, gladiators) {
+    this.name = name.charAt(0).toUpperCase() + name.slice(1).toLowerCase();
+    this.gladiators = []
+  }
+  addGladiator (gladiator){
+    this.gladiators.push(gladiator);
+    if (this.gladiators.length > 2){
+      this.gladiators.pop();
+      console.log(colosseum.gladiators);
+    }
+  }
+  addName (name){
+      this.name.push(name);
+      if (this.name.length > 2){
+        this.name.pop();
+  }
+}
+  fight() {
+    if (this.gladiators[0].weapon === this.gladiators[1].weapon) {
+      this.gladiators.pop();
+      this.gladiators.pop();
+    } else if(this.gladiators[0].weapon > this.gladiators[1].weapon) {
+      this.gladiators.pop();
+    } else {
+      this.gladiators.shift();
+    }
+}
+fightByName () {
```

Comment:
this should deal with the `Gladiators` that are inside of the array at `this.gladiators`